pyproject.toml

```diff
@@ -29,8 +29,6 @@ include = [
     "NOTICE",
     # Poetry by default will exclude these files as they are in .gitignore
     "nautobot/project-static/docs/**/*",
-    # Bundle all of the UI code
-    "nautobot/ui/**/*",
```

Comment:
We're not shipping node_modules or a pre-built UI in the Python wheel (since we can't really build the UI without a config) so no sense bloating it. We get the source files by default because of Poetry including the `nautobot` folder:

```
nautobot/dist/nautobot-2.0.0a2/nautobot/ui on  u/bryanculver-3237-ui2.0-final-build [$]
➜ tree .
.
├── README.md
├── app_imports.js.j2
├── craco.config.js
├── jsconfig-base.json
├── jsconfig.json
├── lib
│   └── nautobot-craco-alias-plugin.js
├── package-lock.json
├── package.json
├── public
│   ├── index.html
│   ├── logo192.png
│   ├── logo512.png
│   ├── manifest.json
│   ├── nautobot_logo.svg
│   └── robots.txt
└── src
    ├── App.js
    ├── components
    │   ├── AppFullWidthComponents.js
    │   ├── AppTab.js
    │   ├── Apps.js
    │   ├── HomeChangelogPanel.js
    │   ├── HomePanel.js
    │   ├── JobHistoryTable.js
    │   ├── Layout.js
    │   ├── LoadingWidget.js
    │   ├── Navbar.js
    │   ├── NotificationPopover.js
    │   ├── ObjectListTable.js
    │   ├── ReferenceDataTag.js
    │   ├── RouterButton.js
    │   ├── RouterLink.js
    │   ├── SidebarNav.js
    │   ├── Table.js
    │   ├── TableItem.js
```

Reply:
This one makes me a bit nervous. Can you build the wheel and show (it's just a tarfile) that it contains all appropriate files still?

Reply:
This is from the untarred tar file.